cmd/wksctl/kubeconfig/kubeconfig.go

```diff
@@ -64,21 +64,38 @@ func init() {
 }
 
 func kubeconfigRun(cmd *cobra.Command, args []string) error {
-	clusterManifestPath, machinesManifestPath, closer, err := manifests.Get(kubeconfigOptions.clusterManifestPath,
-		kubeconfigOptions.machinesManifestPath, kubeconfigOptions.gitURL, kubeconfigOptions.gitBranch, kubeconfigOptions.gitDeployKeyPath,
-		kubeconfigOptions.gitPath)
-	if closer != nil {
-		defer closer()
-	}
-	if err != nil {
-		return err
+	var cpath, mpath string
```

Comment:
the previous names of these arguments `clusterManifestPath` and `machinesManifestPath` were much clearer to me, for whatever it's worth.

Reply:
Hm, you're probably right. I'll rethink the readability (or lack thereof) of this piece.

Reply:
Done - renamed to `clusterPath` and `manifestPath` respectively.